### PR TITLE
Translation zhcn (！！请懂中文的帮忙review)

### PR DESCRIPTION
**Pull request by cliyr**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes

I've been using Node-RED recently and found that some languages are not unified. So I decided to translate English into Chinese simple and did it.

- Translation of side-bar, side-menu, node description, context-menu, etc.

- Corrected previously mistranslated ones (grammar, word-spacing)
Thanks for checking.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
@lkaiy Thank you very much for your contributions.

Do you know any Chinese speaking node-red users in the community who can review your works?

Unfortunately we (I) are not fluent in Chinese and cannot verify your work.

V3.1 release is not too far off so the sooner we can get a review done the more chance your translations will be included in the release.

**Comment by pauldeng** [comment]
Hi all,

Long time node-red user here. 

I think I can be a reviewer. (16 years in Melbourne Australia, native Chinese and near native to Oz English, hehehe)

@Steve-Mcl I am new, can you please point out what is work flow for reviewer if ok?

Thanks.

**Comment by Steve-Mcl** [comment]
> Hi all,
> 
> Long time node-red user here.
> 
> I think I can be a reviewer. (16 years in Melbourne Australia, native Chinese and near native to Oz English, hehehe)
> 
> @Steve-Mcl I am new, can you please point out what is work flow for reviewer if ok?
> 
> Thanks.

Oh that would be great @pauldeng . Thank you very much. The process is fairly simple. Here is a link to a YouTube video it is only 3 minutes but shows you what you need to do. 

https://youtu.be/lSnbOtw4izI

**Review by lkaiy — COMMENTED** [review]
self-inspection pass

**Comment by lkaiy** [comment]
该pr需要除提交者之外的第二个人帮忙review，麻烦请懂中文的朋友帮忙review一下以便完善简体中文翻译，谢谢！！

下面是review的操作方法，视频只有3分钟：
https://youtu.be/lSnbOtw4izI

File: packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json [hunk 1]
```diff
@@ -1,958 +1,1130 @@
...
+    },
+    "state": {
+      "connected": "已连接到服务端: __broker__",
+      "disconnected": "已断开与服务端 __broker__ 的链接",
+      "connect-failed": "与服务端 __broker__ 的连接失败",
+      "broker-disconnected": "客户端连接被 __broker__ 断开: __reasonCode__ __reasonString__"
+    },
+    "retain": "保留",
+    "output": {
+      "buffer": "buffer",
```
Comment (pauldeng) [inline packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json:478]: `"buffer": "Buffer",`

File: packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json [hunk 2]
```diff
@@ -1,958 +1,1130 @@
...
+      "duplicate-path": "同一路径上不能有两个WebSocket侦听器: __path__",
+      "missing-server": "服务器配置缺失",
+      "missing-client": "客户端配置缺失"
+    }
+  },
+  "watch": {
+    "watch": "watch",
+    "label": {
+      "files": "文件",
+      "recursive": "查找所有子文件夹"
```
Comment (pauldeng) [inline packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json:598]: `"recursive": "监视所有子文件夹"`

File: packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json [hunk 3]
```diff
@@ -1,958 +1,1130 @@
...
+  },
+  "file": {
+    "label": {
+      "write": "写入文件",
+      "read": "读取文件",
+      "filename": "文件名",
+      "path": "路径",
+      "action": "操作",
+      "addnewline": "向每个 payload 添加换行符（\\n）",
+      "createdir": "目录不存在自动创建",
```
Comment (pauldeng) [inline packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json:937]: `"createdir": "目录不存在则自动创建",`

**Review by pauldeng — APPROVED** [review]
Appreciate your contributions.

I reviewed and have made some minor change recommendations. Hopefully it will make sense to the community.

I hope more Chinese language users can come and contribute, cheers!

File: packages/node_modules/@node-red/editor-client/locales/zh-CN/editor.json [hunk 4]
```diff
@@ -1,1095 +1,1210 @@
...
+      "close": "关闭",
+      "load": "读取",
+      "save": "保存",
+      "import": "导入",
+      "export": "导出",
+      "back": "后退",
+      "next": "下一个",
+      "clone": "克隆项目",
+      "cont": "继续",
+      "style": "风格",
```
Comment (zoollcar) [inline packages/node_modules/@node-red/editor-client/locales/zh-CN/editor.json:18]:
```suggestion
      "style": "样式",
```

File: packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json [hunk 5]
```diff
@@ -1,958 +1,1130 @@
...
+    }
+  },
+  "comment": {
+    "comment": "注释"
+  },
+  "unknown": {
+    "label": {
+      "unknown": "未知"
+    },
+    "tip": "<p>此节点是您安装，但该节点是Node-RED所不知道的类型。</p><p><i>如果在此状态下部署节点，它的配置将被保留，但是流程将不会启动，直到安装缺少的类型。</i></p><p>有关更多帮助，请参阅信息侧栏</p>"
```
[the remark this answers is not in the document]
Comment (zoollcar) [reply packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json:400]: agree

File: packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json (hunk 3 above)
Comment (zoollcar) [reply packages/node_modules/@node-red/nodes/locales/zh-CN/messages.json:937]: agree

**Review by zoollcar — APPROVED** [review]
The modified one is significantly better than the unmodified

**Pull request merged**